Userland/Libraries/LibJS/Parser.cpp

```diff
@@ -2010,7 +2011,7 @@ NonnullRefPtr<Expression const> Parser::parse_expression(int min_precedence, Ass
             check_for_invalid_object_property(expression);
 
             Associativity new_associativity = operator_associativity(m_state.current_token.type());
-            auto result = parse_secondary_expression(move(expression), new_precedence, new_associativity, forbidden);
```

Comment:
Hmm not sure if this is correct for all cases, but if it doesn't break anything it's fine

Reply:
I ran test262 and it did nothing, good or bad :smile: